### PR TITLE
auto-snooze on a soft proof error after a successful check

### PR DESCRIPTION
**Pull request by maxtaco**
- simple implementation with a reasonable test

### CODE REVIEW

File: go/service/config.go [hunk 1]
```diff
@@ -54,7 +54,11 @@ func (h ConfigHandler) GetExtendedStatus(_ context.Context, sessionID int) (res
 
 	res.Standalone = h.G().Env.GetStandalone()
 	res.LogDir = h.G().Env.GetLogDir()
-	res.Clients = h.G().ConnectionManager.ListAllLabeledConnections()
+
+	// Should work in standalone mode too
+	if h.G().ConnectionManager != nil {
+		res.Clients = h.G().ConnectionManager.ListAllLabeledConnections()
+	}
```
Comment (patrickxb) [inline go/service/config.go]: I assume this is a different bug (status in standalone mode)?  No need to remove, just making sure.

Comment (maxtaco) [inline go/service/config.go]: indeed

**Comment by patrickxb** [comment]
LGTM

**Pull request merged**